src/App.test.js

```diff
@@ -1,9 +1,9 @@
-import React from 'react';
```

Comment:
The changes in this files have been made by eslint so it just include cosmetics.